### PR TITLE
fix(config): add metadata to Nortek devices

### PR DESCRIPTION
**Pull request by disforw**

### CODE REVIEW

**Comment by disforw** [comment]
@zwave-js-bot fix lint

**Comment by AlCalzone** [comment]
@blhoward2 I think this could use your review

File: packages/config/config/devices/0x014f/lb60z-1.json [hunk 1]
```diff
@@ -13,27 +13,55 @@
...
+			"maxNodes": 5,
+			"isLifeline": true
+		}
+	},
 	"supportsZWavePlus": true,
 	"paramInformation": {
 		"1": {
 			"label": "Dim Level Memory",
-			"description": "Dim Settings",
+			"description": "Bulb brightness when turned on",
```
Comment (blhoward2) [inline packages/config/config/devices/0x014f/lb60z-1.json]: I think this description can be the label. No need for both.

File: packages/config/config/devices/0x014f/lb60z-1.json [hunk 2]
```diff
@@ -13,27 +13,55 @@
...
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Disable dim level memory",
+					"label": "Full Brightness",
 					"value": 0
 				},
 				{
-					"label": "Enable dim level memory",
+					"label": "Memory Brightness",
```
Comment (blhoward2) [inline packages/config/config/devices/0x014f/lb60z-1.json]: Change to "Prior brightness level"

File: packages/config/config/devices/0x014f/lb60z-1.json [hunk 3]
```diff
@@ -13,27 +13,55 @@
...
 					"value": 1
 				}
 			]
+		},
+		"9": {
+			"label": "Dim/Bright Step Level",
```
Comment (blhoward2) [inline packages/config/config/devices/0x014f/lb60z-1.json]: Just make this "Dim Step Levels"

File: packages/config/config/devices/0x014f/lb60z-1.json [hunk 4]
```diff
@@ -13,27 +13,55 @@
...
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 99,
+			"defaultValue": 1,
+			"unsigned": true
+		},
+		"10": {
+			"label": "Dim/Bright Speed",
```
Comment (blhoward2) [inline packages/config/config/devices/0x014f/lb60z-1.json]: Make this "Dim Speed"

File: packages/config/config/devices/0x014f/wo15z.json [hunk 5]
```diff
@@ -15,11 +15,11 @@
 	},
 	"paramInformation": {
 		"3": {
-			"label": "Night Light",
+			"label": "Status LED",
```
Comment (blhoward2) [inline packages/config/config/devices/0x014f/wo15z.json]: Change to "LED Indicator"

File: packages/config/config/devices/0x014f/wo15z.json [hunk 6]
```diff
@@ -15,11 +15,11 @@
...
 			"description": "LED mode for attached load",
```
Comment (blhoward2) [inline packages/config/config/devices/0x014f/wo15z.json]: Remove description.

File: packages/config/config/devices/0x014f/ws15z-1.json [hunk 7]
```diff
@@ -20,21 +20,33 @@
 	"paramInformation": {
 		"3": {
 			"label": "Night Light",
```
Comment (blhoward2) [inline packages/config/config/devices/0x014f/ws15z-1.json]: Change to "LED Indicator"

File: packages/config/config/devices/0x014f/ws15z-1.json [hunk 8]
```diff
@@ -20,21 +20,33 @@
...
-			"description": "By default, the LED will turn OFF when the lead attached is turned ON.",
+			"description": "By default, the LED will turn OFF when the load attached is turned ON.",
```
Comment (blhoward2) [inline packages/config/config/devices/0x014f/ws15z-1.json]: Remove description.

File: packages/config/config/devices/0x014f/ws15z-1.json [hunk 9]
```diff
@@ -20,21 +20,33 @@
...
-			"maxValue": 255,
-			"defaultValue": 1,
+			"maxValue": 4,
+			"defaultValue": 0,
 			"unsigned": true,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "On when load is on",
+					"label": "OFF when load is on",
```
Comment (blhoward2) [inline packages/config/config/devices/0x014f/ws15z-1.json]: Don't capitlize the entirety of ON and OFF. Just "On" and "Off". Please carry throughout all files.

**Review by blhoward2 — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x014f/lb60z-1.json (hunk 2 above)
Comment (disforw) [reply packages/config/config/devices/0x014f/lb60z-1.json]: Done

**Comment by AlCalzone** [comment]
@disforw just FYI, the re-review might take a few days

**Review by blhoward2 — DISMISSED** [review]
Approved with minor changes committed

**Review by AlCalzone — APPROVED** [review]
Not sure how I dismissed the other review 🤷🏻‍♂️

**Pull request merged**